docs/user_guide/source/installation/cmake.rst

```diff
@@ -326,97 +34,93 @@ To build ADIOS2:
           SysVShMem: ON
           Endian_Reverse: OFF
 
-    -- Configuring done
-    -- Generating done
-    -- Build files have been written to: /home/chuck/adios2-build
- 
 
-4. Compile using ``make -j`` cores
+Then compile using
+
+.. code-block:: bash
+
+    $ make -j 16
 
-.. code-block:: bash 
-    
-    $ make -j 16 
+Optionally, run the tests:
 
-5. Run tests with make test or ctest
+.. code-block:: bash
 
-.. code-block:: bash 
-    
     $ ctest
     Test project /home/wgodoy/workspace/build
             Start   1: ADIOSInterfaceWriteTest.DefineVar_int8_t_1x10
       1/295 Test   #1: ADIOSInterfaceWriteTest.DefineVar_int8_t_1x10 .........................   Passed    0.16 sec
             Start   2: ADIOSInterfaceWriteTest.DefineVar_int16_t_1x10
       2/295 Test   #2: ADIOSInterfaceWriteTest.DefineVar_int16_t_1x10 ........................   Passed    0.06 sec
             Start   3: ADIOSInterfaceWriteTest.DefineVar_int32_t_1x10
-          
+
           ...
-               
+
             Start 294: ADIOSBZip2Wrapper.WrongParameterValue
     294/295 Test #294: ADIOSBZip2Wrapper.WrongParameterValue .................................   Passed    0.00 sec
             Start 295: ADIOSBZip2Wrapper.WrongBZip2Name
     295/295 Test #295: ADIOSBZip2Wrapper.WrongBZip2Name ......................................   Passed    0.00 sec
-    
+
     100% tests passed, 0 tests failed out of 295
-    
+
     Total Test time (real) =   95.95 sec
-    
 
-5. Install  
 
-.. code-block:: bash 
-    
-    $ make install
+And finally, use the standard invocation to install:
 
+.. code-block:: bash
 
-*******************************
-Build CMake -DVAR=VALUE Options
-*******************************
-
-The following options can be specified with CMake's ``-DVAR=VALUE`` syntax to control which features get enabled or disabled, default option (``ON``/``OFF``) is highlighted:
-
-============================= ========================= ==========================================================================================================================================================================================================================
-CMake VAR Option               Values                     Description                                                                     
-============================= ========================= ==========================================================================================================================================================================================================================
-``ADIOS2_USE_MPI``             **`AUTO`**/``ON``/OFF      MPI or non-MPI (serial) build.                                                                      
```

Comment:
This is my fault for not adding the difference between `Auto` as it has a special meaning, `ON` makes things mandatory and will abort, while `AUTO` with a default `ON` will search for a dependency, but just reported it's not finding it.

Reply:
@williamfgc : I would suggest that this detail will unnecessarily drain mental bandwidth in this location, as they'll learn that when they run `cmake`. But that's a matter of taste; I can change if you want.

Reply:
> as they'll learn that when they run `cmake`

...or blame the user guide, doesn't hurt to add, but I like removing AUTO from here. Another option I never documented is using `ccmake` or `cmake-gui` I've been using `ccmake` recently. `AUTO` is a valid separate third option. We can just explain that AUTO makes the default option not mandatory.